examples/FloatShield/FloatShield_OpenLoop/FloatShield_OpenLoop.ino

```diff
@@ -1,19 +1,55 @@
-#include <FloatShield.h>           // Including the FloatShield library
+/*
+  FloatShield Open Loop
 
-void setup() {                     // Setup - runs only once
-    Serial.begin(9600);            // Begin serial comunication
-    FloatShield.begin();           // Initialise FloatShield
-    FloatShield.calibrate();       // Calibrate FloatShield
+  Performs continuous sensor readings with manual fan speed control.
+
+  The example initializes the FloatShield and performs
+  calibration. Then it reads the current position
+  of the potentiometer and sets the fan speed accordingly.
+  The current position of the potentiometer and current
+  position of the ball is periodically (with specified
+  sampling period) sent out through the serial communication,
+  and can be displayed using Serial Plotter tool of Arduino IDE.
+
+  This code is part of the AutomationShield hardware and software
+  ecosystem. Visit http://www.automationshield.com for more
+  details. This code is licensed under a Creative Commons
+  Attribution-NonCommercial 4.0 International License.
+
+  Created by Gergely Takács and Peter Chmurčiak.
+  Last update: 25.06.2019.
+*/
+
+#include <FloatShield.h>              // Including the FloatShield library
+#include <Sampling.h>                 // Include sampling library
+
+unsigned long Ts = 30000;             // Sampling period in microseconds
+bool nextStep = false;                // Flag for step function
+
+void setup() {                        // Setup - runs only once
+    Serial.begin(250000);             // Begin serial comunication
```

Comment:
Leave it but, just an extra info: AVR can handle up to 2 M.

Reply:
So the arduino can not handle higher baud rate than 2 milion ? If you set it higher it will still be set to 2 milion ? I am not sure if I understood your point correnctly.

Reply:
I guess you wanted me to know, that I can use higer baud rates.

Reply:
Yes, the latter. Just wanted to tell you that you can go up to 2 M.